frontend/app/dashboard/page.jsx

```diff
@@ -1,40 +1,43 @@
 'use client';
-import { useEffect, useState } from 'react';
+import { useEffect } from 'react';
 import { onAuthStateChanged } from 'firebase/auth';
 import { useRouter } from 'next/navigation';
 import { auth } from '../../firebase/config';
 import Cookies from 'js-cookie';
+import { getUser, createUser } from '../../airtable/utils';
+import dashStyle from './Dashboard.module.css';
 
 const Dashboard = () => {
     const router = useRouter();
-    const [ userData, setUserData ] = useState();
 
     useEffect(() => {
-            onAuthStateChanged(auth, user => {
-                if (user) {
-                    const uidVerify = Cookies.get('uid');
-                    if (uidVerify !== user.uid) {
-                        Cookies.remove('uid');
-                        router.push('/')
-                    } else {
-                        setUserData(user);
-                    }
-                } 
-            })
-    }, [])
-    
-    return (
-        <div className='container'>
-            {
-                userData && 
-                <div>
-                    <h1>{ `Welcome to the dashboard` }</h1>
-                    <p>{ `Email: ${userData.email}` }</p>
-                    <p>{ `UID: ${userData.uid}` }</p>
-                </div>
+        const unsubscribe = onAuthStateChanged(auth, async (firebaseUser) => {
```

Comment:
sorry i should have asked last time, why is this function called unsubscribe? Would something like `handleLoginOnPageLoad` or along those lines be more appropriate? Only asking cuz its a key feature, naming is important.